plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -147,38 +184,56 @@ def setup(
         )
         self.sweep_dir = config.hydra.sweep.dir
 
+    def _get_directions(self) -> List[str]:
+        directions: List[str]
+        if isinstance(self.direction, MutableSequence):
+            return [
+                d.name if isinstance(d, Direction) else d for d in self.direction
+            ]
+        elif isinstance(self.direction, str):
+            return [self.direction]
+        return [self.direction.name]
+
+    def _configure_trials(
+        self,
+        trials: List[Trial],
+        search_space: Dict[str, BaseDistribution],
+        fixed_params: Dict[str, Any]
+        ) -> Sequence[Sequence[str]]:
+        overrides = []
+        for trial in trials:
+            for param_name, distribution in search_space.items():
+                trial._suggest(param_name, distribution)
+            for param_name, value in fixed_params.items():
+                trial.set_user_attr(param_name, value)
```

Comment:
Setting fixed parameters as user_attrs in Optuna was previously not done. I think this however makes it more clear for Optuna that this parameter exists and future runs with it not being fixed, might use reuse the information of runs.